### DISCUSSION TITLE
What OS was used in the demo?

### DISCUSSION THREAD

**Discussion by carolgschwend** [spring-projects-experimental/spring-boot-migrator#752] [Q&A]
Could you share what OS and IntelliJ version was used successfully in the [demo](https://www.youtube.com/embed/RKXblzn8lFg).

**Comment by fabapp2** [comment]
Sorry for my very late reply @carolgschwend 
I am on Mac OSX and not sure which exact IntelliJ version this was.
Your question makes me think you had trouble trying the demo?
